sway-lib-std/src/storage.sw

```diff
@@ -323,23 +333,78 @@ impl<V> StorageVec<V> {
     /// fn foo() {
     ///     let five = 5_u64;
     ///     storage.vec.push(five);
-    ///     let retrieved_value = storage.vec.get(0).unwrap();
-    ///     assert(five == retrieved_value);
-    ///     let none_value = storage.vec.get(1);
-    ///     assert(none_value.is_none())
+    ///     assert(five == storage.vec.get(0).unwrap());
+    ///     assert(storage.vec.get(1).is_none())
     /// }
     /// ```
     #[storage(read)]
     pub fn get(self, index: u64) -> Option<V> {
-        let len = get::<u64>(__get_storage_key()).unwrap_or(0);
-
         // if the index is larger or equal to len, there is no item to return
-        if len <= index {
+        if get::<u64>(__get_storage_key()).unwrap_or(0) <= index {
```

Comment:
So I prefer having the temporary variable `len` since it is not really obvious at all that the LHS here is actually the length. This will become clearer after https://github.com/FuelLabs/sway-rfcs/pull/23